### DISCUSSION TITLE
Table filter above improvements

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [filamentphp/filament#4456] [General]
In most of my project I use only two or three filters a table. I was used to the filters being displayed above the table in my October CMS projects. I think there is a lot of white space when using the above table layout option for the table filters.

Is it possible to render the filter form in the active filter indicator section? Just two or three filters? 

Or has anyone a better idea for this use case?

![CleanShot 2022-10-10 at 11 24 19](https://user-images.githubusercontent.com/6506510/194837030-61974b7a-4f8b-4fb4-92f3-73d6f4cf3176.jpeg)

**Comment by zepfietje** [comment]
Now I come to think of it, does it actually make sense to render filter indicators if the filters are always visible (not in a dropdown) anyway?

**Reply by SebastiaanKloos** [reply]
@zepfietje that's right
